proposals/incubation/flatcar.md

```diff
@@ -0,0 +1,354 @@
+
+The list of OS image packages is published with every release. For the current stable it can be viewed here: [https://stable.release.flatcar-linux.net/amd64-usr/current/flatcar_production_image_packages.tx](https://stable.release.flatcar-linux.net/amd64-usr/current/flatcar_production_image_packages.txt)
+
+
+## Roadmap and Issue Tracking
+
+Flatcar uses GitHub extensively for tracking of issues (both raised by the development team and end users), and roadmap items.
+
+Issues are raised and tracked in the [project’s issue tracker](http://issues.flatcar.org/). 
+
+Items on the [project roadmap](https://github.com/orgs/flatcar/projects/7) are derived from requests coming from the community and discussion about which items will be part of the roadmap are done in the public, during office hours, release planning meetings, through Slack and Matrix chats and also in mailing lists forums.
+
+Community contributions are welcomed, and new contributors in particular are encouraged to tackle issues with the tag “good first issue”.
+
+
+## Maintainers and Contributors
+
+The Flatcar codebase has contributions from over 110 people, across both its origins as CoreOS Container Linux and subsequent fork as Flatcar. In the last year, it has had 20 different contributors.
+
+In the last 12 months, there have been 10,446 commits to the project; overall there were more than 49K commits since the beginning of the project. 
+
+The set of maintainers with commit permissions is maintained in the [Maintainers list](https://github.com/flatcar/Flatcar/blob/main/MAINTAINERS.md)
+
+
+## Infrastructure requirements (CI / CNCF Cluster)
+
+Flatcar requires infrastructure for 
+
+* building the distribution, for running system / acceptance tests
+* publishing / serving OS images, with caches for global distribution
+* providing OS updates.
+
+The Flatcar build infrastructure is on Equinix Metal, as are the image servers and caching network (in total ~20 servers of various sizes). System / acceptance tests are run in a variety of vendor clouds (as required to test images specific to those environments) as well as on private cloud infrastructure (like qemu and VMWare) hosted on Equinix Metal’s bare metal servers. Public cloud testing infrastructure is allocated dynamically and freed after testing. Lastly, the update server runs on AWS.
+
+The Equinix Metal infrastructure is currently fully sponsored by Equinix Metal. Upon acceptance of the project into CNCF, Equinix Metal has confirmed its intent to continue that sponsorship under the existing mechanism for sponsored CNCF infrastructure. Where additional infrastructure vendors are willing and able to provide resources to Flatcar as a CNCF project, that would be welcomed.
```

Comment:
Happy to reiterate the Equinix Metal support for the Flatcar build infrastructure, which we fully expect to continue sponsorship of through the CNCF Community Infrastructure Lab (CIL) once this gets accepted.